release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -72,6 +72,11 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-7-web-console"]
 === Web console
 
+[id="ocp-4-7-web-console-localization"]
+==== Web console localization
+
+The web console is now localized and provides language support for global users. The displayed language follows your browser preferences, but you can also select a language if they want to override the browser default. From the *Admin* drop-down menu in the *Administrator* console, select *Language preferences* to update your language setting. Localized date and time is now also supported.
```

Comment:
Should we note the specific languages we support? In 4.7, those will be Japanese, Simplified Chinese, and Korean.

Reply:
Yes, thank you! This is now updated.